pkg/runtime/k0s/init.go

```diff
@@ -0,0 +1,80 @@
+// Copyright © 2022 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package k0s
+
+import (
+	"fmt"
+)
+
+func (k *Runtime) init() error {
+	pipeline := []func() error{
+		k.MergeConfigOnMaster0,
+		k.SSHKeyGenAndCopyIDToHosts,
```

Comment:
I'm not very familiar with k0s, I have a question, why do we need to configure ssh key login?is this necessary？

Reply:
if we don't ssh-keygen id_rsa config: when k0sctl lead to install cluster, it will report:`ERRO [ssh] 172.16.161.63:22: attempt 1 of 60.. failed to connect: stat /root/.ssh/id_rsa: no such file or directory`
if we don't ssh-copy-id: k0sctl will report :`failed to connect: ssh: handshake failed: ssh: unable to authenticate, attempted methods [none publickey], no supported methods remain`